src/components/AlertBox/AlertBox.unit.spec.jsx

```diff
@@ -119,4 +119,47 @@ describe('<AlertBox />', () => {
         backgroundOnly
       />,
     ));
+
+  describe('analytics event', function () {
+    it('should be triggered when link in AlertBox is clicked', () => {
```

Comment:
Since the component is set up with the event handler on the wrapping div, can we have a test that also checks that the event is triggered when a link in the `headline` is clicked?

Reply:
Done!